### ISSUE TITLE
Convert from svg to png: Not all details of emojis are shown in pictures

### ISSUE THREAD

**Issue by Simulacrux** [Ranks/emojione#313]
Hello,

I'm trying to convert the svg files from github to a 24x24 pixel png picture file. 
I'm done this before with the old versions to create a Pidgin smiley theme. After the last update there are several (more or less all) files broken after the convert (missing arms/wheels/eyes/flagparts and so forth). 

I use Inkscape via command line to convert the files:
`inkscape.exe -f "00a9.svg" -e "00a9.png" -w 24 -h 24`

I'm not sure I'm doing something wrong, or it is a problem with the new svg files. Did you change around stuff in the svg alot? Can someone verify this behavier, or must I switch something around in InkScape to get complete pictures back? 
I've done a conversion to 128x128 pixel to clearify the issues in the attached pictures.

Thank you for your help and the nice emojis
Best regards 
Simulacrux

![00a9](https://cloud.githubusercontent.com/assets/12858171/17104524/9d12f2f4-5283-11e6-9b7d-776f17721499.png)
![1f0cf](https://cloud.githubusercontent.com/assets/12858171/17104567/c2b884b0-5283-11e6-99ff-4fd4343910e9.png)
![1f1ec-1f1e7](https://cloud.githubusercontent.com/assets/12858171/17104589/d1b23fd8-5283-11e6-86be-d39ecdc584ff.png)
![1f46b](https://cloud.githubusercontent.com/assets/12858171/17104615/edcf375c-5283-11e6-9147-eebda41fcae2.png)
![1f62b](https://cloud.githubusercontent.com/assets/12858171/17104624/f3bd0856-5283-11e6-85a5-5bc1f938a8c5.png)

**Comment by Simulacrux** [comment]
Hello, 
sorry for bothering you. 
It is/was a bug in the latest stable version of InkScape (https://bugs.launchpad.net/inkscape/+bug/1284409). A new development version fixed the problem. 
So this issue could be closed.

Thank you for the nice emojis
Best regards 
Simulacrux

**Comment by caseyahenson** [comment]
Hi @Simulacrux glad to see you got this working and I apologize for being so late responding. We've occasionally seen this issue in Inkscape as well, however we typically convert uncleaned svgs to png. 

Your mention of the new development version fixing the bug is very important for anybody converting EmojiOne svgs using Inkscape so thank you for finding that.

**Comment by Simulacrux** [comment]
Hi @caseyahenson, 

yeah tried to figure it out in between and got help in the IRC from Inkscape. They brought me to the right direction and showed me where to download the development version. It is a bit tricky.

I've downloaded the Inkscape version from here:
[https://inkscape.org/en/download](https://inkscape.org/en/download) -> Development Versions
The Windows Version is hosted on a seperate Server:
[http://download.tuxfamily.org/inkscape/](http://download.tuxfamily.org/inkscape/)
There I picked the latest available build: Inkscape_0.91+devel_64bit_r15016.7z
This one is working perfectly with the new SVG files.

Glad I could help.
Best regards
Simulacrux
